build.gradle

```diff
@@ -27,18 +27,12 @@ ext {
 dependencies {
     compile 'com.indeed:java-dogstatsd-client:2.0.11'
 
-    if (testWithKafkaClient) {
-      compileOnly "org.apache.kafka:kafka-clients:${kafkaVersion}"
-    } else {
-      compileOnly "org.apache.kafka:kafka_2.11:${defaultKafkaVersion}"
-    }
+    compileOnly "org.apache.kafka:kafka-clients:${kafkaVersion}"
+    compileOnly 'com.yammer.metrics:metrics-core:2.2.0'
     compileOnly 'org.slf4j:slf4j-log4j12:+'
 
-    if (testWithKafkaClient) {
-      testCompile "org.apache.kafka:kafka-clients:${kafkaVersion}"
-    } else {
-      testCompile "org.apache.kafka:kafka_2.11:${kafkaVersion}"
-    }
```

Comment:
Can we keep this part so the CI can help to test on both kafka-clients and kafka core dependencies? So we can test if the library still works on broker side as well.

Reply:
Thanks, I updated it.  Note that I reversed it since the desired behavior is that the published pom won't contain broker dependencies